clarity/evaluator/haspi/eb.py

```diff
@@ -29,7 +29,7 @@ def ear_model(
     level1,
     nchan=32,
     m_delay=1,
-    shift=0.02,
+    shift=None,
```

Comment:
Would it be prudent to add an explanation to the docstring about the bug/issue? This would then be part of the API documentation that is built automatically (see [here](https://claritychallenge.org/clarity/clarity.evaluator.haspi.eb.html#clarity.evaluator.haspi.eb.ear_model)).

A comment for the argument...

```
shift (float): Basal shift of the basilar membrane length. Default is 'None' to align with the original HASPI model derivation (see above note).
```

...and/or a paragraph in the longer description that precedes `Arguments`

```
**NB** - The original HASPI model derivation included a bug where the 'shift' was meant to be set to '0.02' but was never actually applied. For discussion please refer to the discussion in `Issue #105 <https://github.com/claritychallenge/clarity/issues/105>`_ for further details.
```

Reply:
Thanks. That's a good point. I'll address this before merging.